### ISSUE TITLE
Verify VoiceOver works Correctly with App

### ISSUE THREAD

**Issue by sendhil** [wordpress-mobile/WordPress-iOS#2626]
We should do a pass through of the app as it's been a while.

**Comment by koke** [comment]
Moved the comment moderation issue to #4894.

Closing this as it's not very actionable. Accessibility should be an ongoing concern, and we should just open new issues for specific problems.
